### ISSUE TITLE
FOMOD Invalid in 0.0.3 release and Vortex

### ISSUE THREAD

**Issue by Kussie** [pWn3d1337/Skyrim_SpellHotbar2#6]
When trying to install the 0.0.3 release in Vortex it throws a FOMOD invalid error:

```
Invalid XML: The 'https://www.w3.org/2001/XMLSchema-instance:noNamespaceSchemaLocation' attribute is not declared. (Line 2, Pos 64)
The element 'plugin' has incomplete content. List of possible elements expected: 'conditionFlags, typeDescriptor'. (Line 728, Pos 27)
The element 'plugin' has incomplete content. List of possible elements expected: 'conditionFlags, typeDescriptor'. (Line 747, Pos 27)
The element 'plugin' has incomplete content. List of possible elements expected: 'conditionFlags, typeDescriptor'. (Line 753, Pos 27)
The element 'plugin' has incomplete content. List of possible elements expected: 'conditionFlags, typeDescriptor'. (Line 759, Pos 27)
The element 'plugin' has incomplete content. List of possible elements expected: 'conditionFlags, typeDescriptor'. (Line 765, Pos 27)
The element 'plugin' has incomplete content. List of possible elements expected: 'conditionFlags, typeDescriptor'. (Line 771, Pos 27)
The element 'plugin' has invalid child element 'image'. List of possible elements expected: 'description'. (Line 790, Pos 30)
The element 'plugin' has invalid child element 'image'. List of possible elements expected: 'description'. (Line 798, Pos 30)
The element 'plugin' has invalid child element 'image'. List of possible elements expected: 'description'. (Line 806, Pos 30)
```

Likely because Vortex XML validation process is far more strict then that of MO2

**Comment by langfod** [comment]
This seems to work.
[ModuleConfig.xml.txt](https://github.com/user-attachments/files/17499347/ModuleConfig.xml.txt)

**Comment by pWn3d1337** [comment]
I updated my fomod installer python script. will be fixed with 0.0.4

**Comment by pWn3d1337** [comment]
fixed with 0.0.4
